tools/scoring/score.py

```diff
@@ -1882,10 +1882,21 @@ def main():
             print(bl)
             print(lc)
 
-            if (g_args.bestof):
+            if (g_args.bestof != ""):
               # get winner *including non-competitive*
-              winners = grouped[(grouped["rank"] == 1)]
-              print("Best competitive solvers:\n" + winners[["name"]].to_csv())
+              winners = grouped[\
+                          (grouped["rank"] == 1) &\
+                          (grouped["score_correct"] > 0) &\
+                          (grouped["score_error"] == 0)
```

Comment:
What is the motivation for this? Divisions where all solvers were unsound?

Reply:
Yes.  I don't think there are any, but I thought it's a good way of spotting them.

Reply:
Inside the rank == 1 the solvers are ordered by the other criteria of the scoring system (the CPU/wall-clock time mainly, at least in the single-query track).  So the first of those solvers that has rank == 1 is the winner.